gui/src/view.cc

```diff
@@ -185,23 +191,18 @@ void View::redraw(cairo_t* cr)
       // Round to whole pixels
       rect.x = (rect.x/pixelSize) * pixelSize; 
       rect.y = (rect.y/pixelSize) * pixelSize; 
+
+      tx /= pixelSize;
+      ty /= pixelSize;
     }
 
     presentation->redraw(shared_from_this<View>(), cr, rect, zoom);
 
-    if(measurement)
+    // Apply a translation such that (0, 0) is also (0, 0) for the presentation
+    cairo_translate(cr, tx, ty);
+    for(auto renderer : postRenderers)
     {
-      GdkPoint start = presentationPointToWindowPoint(measurement->start);
-      GdkPoint end = presentationPointToWindowPoint(measurement->end);
-      cairo_set_line_width(cr, 1);
-      cairo_set_source_rgb(cr, 0.75, 0, 0); // Dark Red
-      drawCross(cr, start);
-      drawCross(cr, end);
-      cairo_stroke(cr);
-      cairo_set_source_rgb(cr, 1, 0, 0); // Red
-      cairo_move_to(cr, start.x, start.y);
-      cairo_line_to(cr, end.x, end.y);
-      cairo_stroke(cr);
+      renderer->render(shared_from_this<View>(), cr, rect, zoom);
```

Comment:
This feels inconsistent.

`PresentationInterface::redraw()` receives a `rect` argument, the idea being that it should draw whatever that rectangle contains at location `0,0` of the surface.

`PostRenderer::redraw()` receives the same `rect` argument, but this time the idea is that whatever that rectangle contains should be drawn at its "proper" location.

Reply:
Fair enough, mostly did this due to the earlier comments, just reverted it. They both now just get the `rect` the translation for post renderers is gone.